tool/release-notes/running/notes.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
what is the difference between notes.json and latest.json and the notes that are stored under framework/release_notes? Why are we storing two copies?

Reply:
I am hoping to move all of the release notes stuff up into the tool folder, so I am hoping to delete the release note stuff from that other location